### DISCUSSION TITLE
support target `-linux-amd64` as alias to `-linux-x64`

### DISCUSSION THREAD

**Discussion by viceice** [yao-pkg/pkg#24] [Ideas]
I like to use `TARGETARCH` variable of docker buildx to copy only the relevant file to the docker image.
The `TARGETARCH` is `amd64` for `x86_64` and `arm64` for `aarch64`.

- https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
- https://github.com/containerbase/base/issues/2079

**Comment by robertsLando** [comment]
would you like to submit a pr?

**Reply by viceice** [reply]
sure, do yo have some hints where to look at?

**Reply by robertsLando** [reply]
I think you should add the alias support in this function: https://github.com/yao-pkg/pkg/blob/7f95e50ea90efe9776b5c1822d38780b40433750/lib/index.ts#L53

**Reply by robertsLando** [reply]
Remember to add a test :)

**Reply by viceice** [reply]
will look at it tomorrow

**Reply by viceice** [reply]
shouldn't this be added here?
https://github.com/yao-pkg/pkg-fetch/blob/c7dff099190a393a27f24880b154dbc87acb2ede/lib/system.ts#L80

**Reply by robertsLando** [reply]
Yeah thought that was just used to parse local arch but seems is used also [here](https://github.com/yao-pkg/pkg-fetch/blob/c7dff099190a393a27f24880b154dbc87acb2ede/lib/index.ts#L99) so it should be good :)
